presto-spark-base/src/main/java/com/facebook/presto/spark/execution/PrestoSparkRemoteSourceFactory.java

```diff
@@ -36,38 +36,38 @@
         implements RemoteSourceFactory
 {
     private final PagesSerde pagesSerde;
-    private final Map<PlanNodeId, Iterator<PrestoSparkMutableRow>> shuffleInputs;
-    private final Map<PlanNodeId, Iterator<PrestoSparkSerializedPage>> broadcastInputs;
+    private final Map<PlanNodeId, Iterator<PrestoSparkMutableRow>> rowInputs;
```

Comment:
I prefer the old variable name (`shuffleInputs` and `broadcastInputs`). Otherwise, it's not immediately clear why we need two type of inputs.

Reply:
I had to add third input type, `localInputs`. This is needed to provide inputs for the fragment that is running on the driver (COORDINATOR_ONLY fragment). Now `broadcastInputs` is a superset of both, local and broadcast inputs. That's why I decided to make this rename, as currently there's no 1-1 mapping.

Reply:
@arhimondr : What about `remoteBroadcastInputs`? 😃 

Also I would still argue, even with slightly confusion, `shuffleInputs` and `broadcastInputs` are better than `rowInputs` and `pageInputs`. `rowInputs` and `pageInputs` doesn't contain any extra information -- the fact that the input type is Page/Row can be read from its type.

Reply:
It feels like at the `RemoteSourceFactory` level of abstraction it doesn't matter where the inputs come from. What matters is the type of the input. Whether it supplies rows or pages. In theory, in the future, shuffle input could also supply pages (for example if the downstream stage is single node and all rows are written to the same partition there's no need to convert pages to rows).